README.md

```diff
@@ -60,111 +44,37 @@ iwasm VM core
 - [Multi-value](https://github.com/WebAssembly/multi-value), [Tail-call](https://github.com/WebAssembly/tail-call), [Shared memory](https://github.com/WebAssembly/threads/blob/main/proposals/threads/Overview.md#shared-linear-memory)
 
 ### Supported architectures and platforms
-
-The iwasm supports the following architectures:
-
+The iwasm supports the following architectures:  
```

Comment:
Had better change to `VMcore` or `WAMR`?

Reply:
yes

Reply:
updated